dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -303,6 +303,11 @@ public void manualPublish(final List<String> args) {
             container.setToolname(toolname);
             container.setPath(Joiner.on("/").skipNulls().join(registry, namespace, name));
 
+            // Check that tool has at least one default path
+            if ((cwlPath == null || cwlPath.equals("")) && (wdlPath == null || wdlPath.equals(""))) {
```

Comment:
This can be simplified with https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/base/Strings.html#isNullOrEmpty(java.lang.String)